### ISSUE TITLE
Better handling of member list presences, roles

### ISSUE THREAD

**Issue by MaddyUnderStars** [MaddyUnderStars/Fosscord-BD#2]
Currently, upon GUILD_MEMBER_LIST_UPDATE the plugin will dispatch member list update to the client, which I would expect to be enough to update the roles, presences, etc of the members in the list. Apparently not. I'm probably just sending the wrong keys/data structure internally. As such, the current solution for presences ( and roles, on a different branch ) is to also dispatch internally guild member role add and presence update events.

Anyway, long story short: Figure out what the hell data structure discord expects for the guild member list update event
